Traceback (most recent call last):
  File "/home/alvaro/anaconda3/envs/pymc_env/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/alvaro/anaconda3/envs/pymc_env/lib/python3.11/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alvaro/anaconda3/envs/pymc_env/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alvaro/anaconda3/envs/pymc_env/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/alvaro/anaconda3/envs/pymc_env/lib/python3.11/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/alvaro/anaconda3/envs/pymc_env/lib/python3.11/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/alvaro/anaconda3/envs/pymc_env/lib/python3.11/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

from sklearn.linear_model import LogisticRegression
from sklearn.svm import SVC

from sklearn import metrics
from sklearn.metrics import ConfusionMatrixDisplay

import lightgbm as lgbm

from imblearn import over_sampling
from imblearn import under_sampling
from imblearn import combine
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[1], line 11[0m
[1;32m      8[0m [38;5;28;01mfrom[39;00m [38;5;21;01msklearn[39;00m [38;5;28;01mimport[39;00m metrics
[1;32m      9[0m [38;5;28;01mfrom[39;00m [38;5;21;01msklearn[39;00m[38;5;21;01m.[39;00m[38;5;21;01mmetrics[39;00m [38;5;28;01mimport[39;00m ConfusionMatrixDisplay
[0;32m---> 11[0m [38;5;28;01mimport[39;00m [38;5;21;01mlightgbm[39;00m [38;5;28;01mas[39;00m [38;5;21;01mlgbm[39;00m
[1;32m     13[0m [38;5;28;01mfrom[39;00m [38;5;21;01mimblearn[39;00m [38;5;28;01mimport[39;00m over_sampling
[1;32m     14[0m [38;5;28;01mfrom[39;00m [38;5;21;01mimblearn[39;00m [38;5;28;01mimport[39;00m under_sampling

[0;31mModuleNotFoundError[0m: No module named 'lightgbm'

